### PR TITLE
{cae}[intel/2019a] OpenFOAM v2.2.x

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.2.x-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name = 'OpenFOAM'
+local_commit = '1f35a0f'
+version = '2.2.x'
+
+homepage = 'http://www.openfoam.com/'
```
Comment (Micket) [inline easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.2.x-intel-2019a.eb]: https

File: test/easyconfigs/easyconfigs.py [hunk 2]
```diff
@@ -453,6 +453,28 @@ def check_sanity_check_paths(self, changed_ecs):
...
+        http_regex = re.compile('http://[^"\'\n]*', re.M)
+
+        failing_checks = []
+        for ec in changed_ecs:
+            ec_fn = os.path.basename(ec.path)
+            for http_url in http_regex.findall(ec.rawtxt):
+                https_url = http_url.replace('http://', 'https://')
+                try:
+                    https_url_works = bool(urlopen(https_url))
+                except HTTPError, URLError:
```
Comment (Micket) [inline test/easyconfigs/easyconfigs.py]: fix/rebase on develop please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3122.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/72b6ae199447dd58cf345fc975d4203f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2671.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/11fef0e7cd07bc3c8a15b36323f9e99a for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2094.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/16516f7c9e62b44797e1592e460d9977 for a full test report.

**Pull request merged**